projects/meta_cl/networks/anml_networks.py

```diff
@@ -77,7 +77,7 @@ def conv_block(cls, in_channels, out_channels, kernel_size, stride, padding):
                 in_channels=in_channels, out_channels=out_channels,
                 kernel_size=kernel_size, stride=stride, padding=padding
             ),
-            nn.BatchNorm2d(num_features=out_channels),
+            nn.InstanceNorm2d(num_features=out_channels, affine=True),
```

Comment:
We can drop the train=True in the eval function, that is no longer required, and can bring some unexpected side effects in the future (for example if we include dropout in one of these networks). InstanceNorm2d has `truck_running_stats=False` as default.